frontend/public/index.html

```diff
@@ -8,11 +8,9 @@
     <title>Tectonic Console</title>
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="<%= require('./lib/mochi/img/favicons/tectonic/apple-touch-icon-114x114.png') %>">
-    <link rel="apple-touch-icon-precomposed" sizes="114x114" href="<%= require('./lib/mochi/img/favicons/tectonic/apple-touch-icon-144x144.png') %>">
-    <link rel="apple-touch-icon-precomposed" sizes="72x72" href="<%= require('./lib/mochi/img/favicons/tectonic/apple-touch-icon-72x72.png') %>">
-    <link rel="apple-touch-icon-precomposed" sizes="57x57" href="<%= require('./lib/mochi/img/favicons/tectonic/apple-touch-icon-57x57.png') %>">
-    <link rel="shortcut icon" href="<%= require('./lib/mochi/img/favicons/tectonic/favicon-96x96.png') %>">
+    <link rel="shortcut icon" href="<%= require('./lib/mochi/img/favicons/openshift/favicon.png') %>">
+    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="<%= require('./lib/mochi/img/favicons/openshift/apple-touch-icon-precomposed.png') %>">
+    <link rel="mask-icon" href="<%= require('./lib/mochi/img/favicons/openshift/mask-icon.svg') %>" color="#DB242F">
```

Comment:
We should consider if we want to keep supporting both Tectonic and OpenShift branding in master for the near term. It is possible to have different favicons with some logic since this is a Go template.

Reply:
And if we're thinking about vanilla kube support, we probably want two different sets...

Reply:
I moved these into /imgs/ , but didn't delete the old versions.  Will need help with Go if we go down that route.